release/_publishing.sh

```diff
@@ -36,19 +36,47 @@ function init_gcs {
 }
 
 function update_latest_in_bundles {
-    local file_path="${BASE_DIR}/bundles.yml"
+	local product_version_key=${_PRODUCT_VERSION%-*}
 
-    if (yq eval ".quarterly | has(\"${_PRODUCT_VERSION}\")" "${file_path}" | grep -q "true")
-    then
-        lc_log INFO "The ${_PRODUCT_VERSION} product version was already published."
+	local file_path="${BASE_DIR}/bundles.yml"
```

Comment:
@kiwm, please declare the variables according to their usage and sort them alphabetically if possible, like:
```
local file_path="${BASE_DIR}/bundles.yml"
local product_version_key=${_PRODUCT_VERSION%-*}
```